test/e2e/autoscale_custom_test.go

```diff
@@ -76,6 +98,26 @@ func TestUpDownCustomMetric(t *testing.T) {
 	assertCustomHPAAutoscaleUpToNumPods(ctx, targetPods, time.After(scaleUpTimeout), true /* quick */)
 }
 
+func TestUpDownCustomMetricTemplate(t *testing.T) {
```

Comment:
I think this is redundant as we have the multi revision test where we test the template feature.

Reply:
removed